dev/com.ibm.ws.session.cache_fat_infinispan_container/fat/src/com/ibm/ws/session/cache/fat/infinispan/container/FATSuite.java

```diff
@@ -71,6 +72,12 @@ public class FATSuite extends TestContainerSuite {
                                                         "com.ibm.ws.session.cache.fat.infinispan.container.timeoutServerB")
                                             .fullFATOnly())
                             .andWith(new JakartaEE10Action()
+                                            .forServers("com.ibm.ws.session.cache.fat.infinispan.container.server",
+                                                        "com.ibm.ws.session.cache.fat.infinispan.container.serverA",
+                                                        "com.ibm.ws.session.cache.fat.infinispan.container.serverB",
+                                                        "com.ibm.ws.session.cache.fat.infinispan.container.timeoutServerA",
+                                                        "com.ibm.ws.session.cache.fat.infinispan.container.timeoutServerB"))
+                            .andWith(FeatureReplacementAction.EE11_FEATURES()
```

Comment:
For consistency, I think we should either replace `FeatureReplacementAction.EE11_FEATURES()` with `new JakartaEE11Action();` or update the others to use the static method. For example, change `new JakartaEE10Action` to `FeatureReplacementAction.EE10_FEATURES()`

Reply:
Thanks. FYI. they purposely make JakartaEE11Action() protected. 

    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
    // This constructor is purposely not public to force people to use the JakartaEEAction class and                 //
    // the FeatureReplacementAction.EE11_FEATURES() method instead of referencing this class directly                //
    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
    protected JakartaEE11Action() {

Reply:
With the latest test framework you can simplify this quite a bit. See https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.el.3.0_fat/fat/src/com/ibm/ws/el/fat/FATSuite.java as an example.

1) You're currently running both EE10 and EE11 in lite mode, we should only run the latest repeat in lite mode:  https://github.com/OpenLiberty/open-liberty/wiki/FAT-Tests#repeating-fat-tests-for-different-featurejee-levels
2) You list all the servers in the bucket, I don't think it is necessary in this case.

Reply:
Updated, but I have list all the server else no server was found.